pkg/command/plugin.go

```diff
@@ -442,6 +446,7 @@ func newCleanPluginCmd() *cobra.Command {
 	var cleanCmd = &cobra.Command{
 		Use:   "clean",
 		Short: "Clean the plugins",
+		Long:  `Clean or remove all installed plugins from the system`,
```

Comment:
Because I can't help being difficult, could you make this a normal string? (using `""`)

Also, we shouldn't say "clean or remove", it makes it sound like we're not sure that the command does.  How about:

```suggestion
		Long:  "Remove all installed plugins from the system",
```